docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+  - set the swh_type to OTHER.  If the tank_type is an identifiable type, swh_type will be overwritten to that type: `swh_type = "OTHER"`
+  - if the tank_type is CONSUMER_INSTANTANEOUS, COMMERCIAL_INSTANTANEOUS, or RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS, set the swh_type to INSTANTANEOUS and go to rule assertion: `if tank_type in ["CONSUMER_INSTANTANEOUS", "COMMERCIAL_INSTANTANEOUS", "RESIDENTIAL_DUTY_COMMERCIAL_INSTANTANEOUS"]: swh_type = "INSTANTANEOUS"`; GO TO RULE_ASSERTION
+
+  - get the fuel type of the equipment: `fuel_type = shw_eq.heater_fuel_type`
+  - if the tank_type is CONSUMER_STORAGE or COMMERCIAL_STORAGE, set the swh_type to STORAGE for the fuel type: `if tank_type in ["CONSUMER_STORAGE", "COMMERCIAL_STORAGE"]:`
+    - set the swh_type based on tank_type and fuel type: `if fuel_type == "ELECTRICITY": swh_type = "ELECTRIC_RESISTANCE_STORAGE`
+    - `elif fuel_type == "NATURAL_GAS": swh_type = "GAS_STORAGE`
+    - `elif fuel_type == "FUEL_OIL": swh_type = "OIL_STORAGE`
+  - if the draw_pattern is given, get the draw pattern: `if shw_eq.draw_pattern != nil: draw_pattern = shw_eq.draw_pattern`
+  - otherwise, determine the draw pattern based off the first_hour_rating of the shw_eq: `else:`
+    - First get the first hour rating (gallons): `first_hour_rating = shw_eq.first_hour_rating`
+    - if the first_hour_rating is less than 18 gallons, draw_pattern is "VERY SMALL": `if first_hour_rating < 18: draw_pattern = "VERY_SMALL"`
+    - otherwise if the first_hour_rating is less than 51 gallons, draw pattern is LOW: `elif first_hour_rating < 51: draw_pattern = "LOW"`
+    - otherwise if the first_hour_rating is less than 75 gallons, draw pattern is MEDIUM: `elif first_hour_rating < 75: draw_pattern = "MEDIUM"`
+    - otherwise draw pattern is HIGH: `else: draw_pattern = "HIGH"`
+  - get the capacity of the SHW system (make sure to convert to btu/hr): `input_capacity = swh_eq.input_power`
+  - get the volume of the tank (make sure to convert to gallons): `storage_volume = swh_eq.distribution_system.tank.storage_capacity`
+  - calculate the capacity_per_volume based on the input_capacity and the storage_volume: `capacity_per_volume = shw_eq.rated_capacity / storage_volume`
+  - get the efficiency metric values: `efficiency_metric_values = swh_eq.efficiency_metric_values`
+  - get the efficiency metric types: `efficiency_metric_types = swh_eq.efficiency_metric_types`
+  - create a dictionary of the efficiency data: `efficiency_metric_dict = dict(zip(efficiency_metric_types, efficiency_metric_values))
+  - get the thermal efficiency: `thermal_efficiency = efficiency_metric_dict["THERMAL_EFFICIENCY"]`
+  - get the uniform energy factor: `uniform_energy_factor = efficiency_metric_dict["UNIFORM_ENERGY_FACTOR"]`
+  - what follows is logic that implements Table 7.8 - this table has multiple criteria, so I think it's simpler to implement with if / else logic than a table lookup
```

Comment:
Do we need to review this logic if it is just implementing Table 7.8 lookups? It is a lot of lines and I think we should be able to rely on unit tests to verify everything works as intended.